httpcore5/src/main/java/org/apache/hc/core5/http/io/entity/SerializableEntity.java

```diff
@@ -59,29 +62,35 @@
      *        stored in an internal buffer
      * @throws IOException in case of an I/O error
      */
-    public SerializableEntity(final Serializable ser, final boolean bufferize) throws IOException {
-        super();
+    public SerializableEntity(
+            final Serializable ser, final boolean bufferize, final ContentType contentType,
+            final String contentEncoding) throws IOException {
+        super(contentType, contentEncoding);
         Args.notNull(ser, "Source object");
         if (bufferize) {
             createBytes(ser);
+            this.objRef = null;
```

Comment:
I don't understand what's happening here. Why are you nulling this field out after `createBytes` has assigned it?

Reply:
@rschmitt Neither do it. I did not author this class.